File: Python/beecrowd_1115.py
ans = ("primeiro","segundo","terceiro","quarto")
while(1):
    x, y = map(int,input().split())
    if (abs(x) > 0 and abs(y) > 0):
        i = (0 if y > 0 else 3) if (x > 0) else (2 if y < 0 else 1)
        print("%s" %ans[i])
    else:
        break

File: Cpp/beecrowd_1115.cpp
#include <stdio.h>
#include <string>

int main() {
    
    std::string ans[] = {"primeiro","segundo","terceiro","quarto"};
    int x, y, i;
    while(1){
        scanf("%d %d", &x, &y);
        if (abs(x) > 0 && abs(y) > 0) {
            i = (x > 0) ? (y > 0 ? 0 : 3) : (y < 0 ? 2 : 1);
            printf("%s\n",ans[i].c_str());
        }else {
            break;
        }    
    }
    return 0;
}
